Deepanshu_Balani/project1 profile/profile_new.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>myprofile</title>
+    <link rel="stylesheet" type="text/CSS" href="./profiles.css" />
+    <link
+      rel="stylesheet"
+      type="text/CSS"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      <link
+      href="https://fonts.googleapis.com/css?family=Gelasio&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width,initial-state=1"/>
+<link href="css/bootstrap.css "rel="stylesheet"/>
+<script src="js/jquery-3.2.1.min.js"></script>
+<script src="js/bootstrap.js"></script>
+<style type="text/css">
+    <style>
+      h2:hover ul {
+        display: block;
+        background-color: turquoise;
+        
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container-fluid back">
+      
+         <div class="row">
+            <div class="col-lg-1 col-md-4 col-sm-12 col-xs-12"></div>
```

Comment:
this div will occupy only 1 column on large screen, not sure it will look visually good there
Check for responsiveness

Reply:
the page is made responsive now ,i have put col-lg-4 instead of col-lg-1